SU-HSDO/stanford_mrc
# MRCD8-247 added search icon and expandable search functionality

# READY FOR REVIEW

# Summary
- Added search (svg) icon
- Added expandable search (targeting Drupal's search block)
- Minor cleanup of other base files (mostly comments and spacing)

# Needed By (Date)
- End of sprint (3/1)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the search placed in the header of the site is now a circle with a magnifying glass
When you click/focus on the magnifying glass, the search input should expand

NOTE: the work to place items in the correct order, etc. will be completed as part of a future ticket; currently each item in the header is being attended to one-by-one.

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-247

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
